src/components/CustomTable/StyledTableBody/StyledTableBody.js

```diff
@@ -0,0 +1,43 @@
+import { TableBody, TableRow, makeStyles } from "@material-ui/core"
+import { get } from "lodash"
+
+import StyledTableCell from "../StyledTableCell"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    "&:nth-of-type(even)": {
+      backgroundColor: theme.palette.secondary.main,
+    },
+    "&$hover:hover": {
+      backgroundColor: theme.palette.primary.main,
+      cursor: "pointer",
+    },
+  },
+  //Produce warning if removed
+  hover: {},
```

Comment:
What warning is this giving?

Reply:
It produces this warning below.

`Warning: [JSS] Could not find the referenced rule "hover" in "makeStyles".`

This also affects the styling under the hover state
```
root: {
    "&$hover:hover": {
      backgroundColor: theme.palette.primary.main,
      cursor: "pointer",
    },
  },
hover: {},
``` 

It seems that to apply the style on the hover state on that component, it is required for us to do code like that to override the default class
```
.MuiTableRow-root.MuiTableRow-hover:hover {
    background-color: rgba(0, 0, 0, 0.04);
}
```

Reply:
You can change your CSS to look like this:

```
![image](https://user-images.githubusercontent.com/23342131/227897624-d61f0d58-63e2-4206-bbb8-50aac76ab933.png)

```

And remove the `hover` prop from the `TableRow` component